environment_linux.yml

```diff
@@ -38,7 +39,6 @@ dependencies:
   - tabulate
   - pip:
     - pydegensac
-    - argoverse @ git+https://github.com/argoai/argoverse-api.git@master
```

Comment:
@johnwlambert I am removing the argoverse as it forces install of opencv 4.5.3, and the USAC in that release does not work (it says it wants Lapack/Eigen installed). 

This is the shortest fix i could think ok. A long term fix will be to figure out if argoverse can work with 4.5.0, and to prevent the install of a duplicate version of opencv by argoverse pip install

Reply:
i wonder why argoverse is forcing this. argoverse doesnt pull a specific opencv version, but just pulls
```
        "opencv-python>=4.1.0.25",
```
from pypi. i guess the conda `opencv=4.5.0` is not compatible with pypi `opencv-python>=4.1.0.25`? i wonder why that is.

Reply:
Yep that is correct. Pypi and conda distributions of opencv go by different names. Conda's distribution installs libopencv, opencv, and py-opencv.